frame/balances/dispatchables/transfer.go

```diff
@@ -227,8 +227,8 @@ func tryMutateAccount(who types.Address32, f func(who *types.AccountData, bool b
 
 	r := result.Value.(sc.VaryingData)
 
-	dustCleaner := r[1].(DustCleanerValue)
-	dustCleaner.Drop()
+	//dustCleaner := r[1].(DustCleanerValue)
```

Comment:
don't we need this?

Reply:
Currently, it emits an event every time, which should not be the case. I'll add a TODO to fix it when the functionality is refactored.